gprofiler/profilers/python_ebpf.py

```diff
@@ -256,9 +256,14 @@ def _terminate(self) -> Optional[int]:
             self.process.terminate()  # okay to call even if process is already dead
             code = self.process.wait()
             self.process = None
+
```

Comment:
Can comment on the line but change above to use `self.is_running` instead of `        if self.process is not None:
`

Reply:
Bleh, mypy then whines, it doesn't follow the validty of the assert through functions :/